.github/workflows/oss-ent-merge-trigger.yml

```diff
@@ -0,0 +1,25 @@
+name: Trigger OSS to Enterprise Merge
```

Comment:
Is this entire file part of this PR? To me seems out of context. Could we have a separate PR just for the GH workflow?